testapp/exam/settings.py

```diff
@@ -18,3 +18,11 @@
 # reason set this to the root you have to serve at.  In the above example
 # host.org/foo/exam set URL_ROOT='/foo'
 URL_ROOT = ''
+
+code_evaluators = {"python": "python_code_evaluator.PythonCodeEvaluator",
```

Comment:
I tried using absolute paths - But since these evaluator paths will be imported within `code_server.py` and eventually `language_registry.py` which are located within `exam` directory, hence such absolute paths don't work.